### DISCUSSION TITLE
Bit-error-rate (BER)

### DISCUSSION THREAD

**Discussion by Deu82** [InterDigitalInc/AdvantEDGE#302] [Q&A] [answered]
Hi @roymx ,

I was wondering if we could get bit-error-rate (BER) from a scenario in AdvantEDGE? I know that, we can obtain latency information (i.e. response time delay), say for example from a scenario with 2 UEs and one PoA. 

Is there any way to get the BER value from such a scenario?

Thanks a lot for your help!

**Comment by roymx** [comment, marked as answer]
Hello @Deu82 ,

Unfortunately, we do not measure BER on the platform, so it explains why it is not available in the time-series database if that is what you are asking.

In AdvantEDGE, I believe BER would mainly be caused by the Packet Loss settings present on a path between two scenario nodes.
If you need to understand how the packet loss impacts a scenario data path, you can look at [this section](https://interdigitalinc.github.io/AdvantEDGE/docs/overview/overview-architecture/#network-characteristics)
